internal/services/network/express_route_connection_resource.go

```diff
@@ -231,6 +238,10 @@ func resourceExpressRouteConnectionRead(d *pluginsdk.ResourceData, meta interfac
 		d.Set("authorization_key", props.AuthorizationKey)
 		d.Set("enable_internet_security", props.EnableInternetSecurity)
 
+		if props.EnablePrivateLinkFastPath != nil {
+			d.Set("enable_private_link_fast_path", props.EnablePrivateLinkFastPath)
+		}
```

Comment:
We could simplify this to 
```suggestion
		d.Set("enable_private_link_fast_path", pointer.From(props.EnablePrivateLinkFastPath))
```

Reply:
I've refactored this in https://github.com/hashicorp/terraform-provider-azurerm/pull/25596/commits/3e0b03326152b1f8ed6bfaa6e585ad3467234216